tests/components/FilterSearch.stories.tsx

```diff
@@ -77,3 +77,11 @@ DropdownSectioned.play = ({ canvasElement }) => {
   const canvas = within(canvasElement);
   userEvent.type(canvas.getByRole('textbox'), 'name');
 };
+
+export const NoLabel = (args: FilterSearchProps) => {
+  return (
+    <AnswersHeadlessContext.Provider value={generateMockedHeadless(mockedHeadlessState)}>
+      <FilterSearch searchFields={searchFields} {...args} />
+    </AnswersHeadlessContext.Provider>
+  )
```

Comment:
oh for these you can run `npx eslint . --fix` and it'll autofix simple things like this